pkg/authorization/workspace_content_authorizer.go

```diff
@@ -112,10 +113,15 @@ func (a *workspaceContentAuthorizer) Authorize(ctx context.Context, attr authori
 	}
 
 	// always let logical-cluster-admins through
-	if isUser && sets.NewString(attr.GetUser().GetGroups()...).Has(bootstrap.SystemLogicalClusterAdmin) {
+	if isUser && groups.Has(bootstrap.SystemLogicalClusterAdmin) {
 		return DelegateAuthorization("logical cluster admin access", a.delegate).Authorize(ctx, attr)
 	}
 
+	// always let external-logical-cluster-admins through
+	if isUser && groups.Has(bootstrap.SystemExternalLogicalClusterAdmin) {
+		return DelegateAuthorization("external logical cluster admin access", a.delegate).Authorize(ctx, attr)
+	}
+
```

Comment:
this we don't want. Why is it necessary that the external logical cluster admin skips the existence test? It shouldn't.

Reply:
I moved it into the switch with the other checks